src/cli.ts

```diff
@@ -0,0 +1,254 @@
+import * as cli from '@rushstack/ts-command-line';
+import { MigrateDownOptions, MigrateUpOptions, Umzug } from './umzug';
+
```

Comment:
With your other suggestion, the type system now protects against this, so I'll skip this since the constructor is internal so it's effectively unreachable code.